tools/scripts/yaml_to_excel.py

```diff
@@ -153,7 +153,7 @@ def create_security_requirements_sheet(wb):
         write_header(ws, category_title, mas_styles.MASVS_COLORS[group_id])
         set_columns_width(ws)
 
-        status_cells = 'H8:L400'
+        status_cells = 'I13:I400'
```

Comment:
Another fix so that we can also mark the first control, not only tests.
```suggestion
        status_cells = 'I11:I400'
```